src/components/table/UITable.vue

```diff
@@ -361,13 +360,13 @@ export default defineComponent({
     setup(props, ctx) {
         const chevronIcon = getIcon('chevron');
         let styleTagId = '';
-        const itemPerPageOptions = [
+        const itemPerPageOptions = markRaw([
```

Comment:
What is the intention of `markRaw` being used for `itemPerPageOptions`?

Reply:
Performance improvement (and perhaps readibility)
https://vuejs.org/api/reactivity-advanced.html#markraw
> ...allow you to selectively opt-out of the default deep reactive/readonly conversion and embed raw, non-proxied objects in your state graph

Reply:
Oh I know what it does, but I'm confused about the performance aspect for a small, hard-coded, never-changing internal array.

No downside to having it there, just can't see the upside in this particular instance.